src/theme/colors.ts

```diff
@@ -51,6 +51,10 @@ const commonColors = {
     700: '#C74543',
     800: '#AE3C3B',
     900: '#923231'
+  },
+  darkTeal: {
```

Comment:
Yah not sure where darkTeal came from, but any positive action should be `green.500` just to make sure we aren't using different colors in different places.